packages/next/build/webpack/loaders/next-swc-loader.js

```diff
@@ -88,6 +88,9 @@ async function loaderTransform(parentTrace, source, inputSourceMap) {
   const swcSpan = parentTrace.traceChild('next-swc-transform')
   return swcSpan.traceAsyncFn(() =>
     transform(source, programmaticOptions).then((output) => {
+      if (output.eliminatedPackages && this.eliminatedPackages) {
+        this.eliminatedPackages.push(...JSON.parse(output.eliminatedPackages))
```

Comment:
It will be deduped before report https://github.com/vercel/next.js/pull/35404/files#diff-9c3b824be9edf62be396f869dc7456bdc4ce8822347e40c901eff9780c7246adR158

Reply:
I also think it's better to make it a Set and dedupe directly during capturing. Why accumulating a large list and deduplicating at the end when you could directly dedupe it.